geode-connectors/src/main/resources/META-INF/schemas/geode.apache.org/schema/jdbc/jdbc-1.0.xsd

```diff
@@ -41,19 +41,6 @@ XML schema for JDBC Connector Service in Geode.
     <xsd:element name="connector-service">
```

Comment:
We still have jdbc-1.0.xsd with connector-service element. However, we deleted the namespace and further down in this review we deleted processing for connector-service.

Reply:
(Looks like my comments got reordered by github so further down might be up)

Reply:
The namespace we deleted had to do with the "connection" which was deleted not the connector-service. I could not find were we deleted processing for connector-service; it should have just been for connection. Let me know if this should not have been resolved.